src/phase0/block_processing.rs

```diff
@@ -364,8 +364,9 @@ pub fn process_deposit<
 
     let public_key = &deposit.data.public_key;
     let amount = deposit.data.amount;
+    let cloned_validators = state.validators.clone();
```

Comment:
same question here

Reply:
For this, if i don't clone, mutable borrow occurs when i try to iterate over the validators and put them into a HashSet